models/articles.model.js

```diff
@@ -1,7 +1,38 @@
 const db = require("../db/connection")
 
 
-function fetchArticleID (article_id) {  
+
+exports.fetchArticles = () => {
+    return db.query(
+`SELECT   
+    articles.article_id, 
+    articles.title, 
+    articles.topic, 
+    articles.author, 
+    articles.created_at, 
+    articles.votes, 
+    articles.article_img_url,
+    COUNT(comments.article_id) AS comment_count
```

Comment:
in order to make comment_count a number rather than a string, simply add `::INT` before `AS`